arch/arc/Kconfig

```diff
@@ -280,6 +280,15 @@ config CACHE_FLUSHING
 	  If the d-cache is present, set this to y.
 	  If the d-cache is NOT present, set this to n.
 
+config ARC_EXCEPTION_STACK_SIZE
```

Comment:
good, but grepping through the code I find that we have:
- ARMV7_EXCEPTION_STACK_SIZE (for arm)
- EXCEPTION_STACK_SIZE (for x86)

and now we are adding one for ARC, can we just have one top-level kconfig for all architectures and call it EXCEPTION_STACK_SIZE?

Reply:
@nashif Most architectures don't have a separate exception stack

Reply:
That's only Cortex-R @nashif